packages/preset-attributify/README.md

```diff
@@ -177,12 +177,10 @@ declare module 'solid-js' {
 ### Svelte & SvelteKit
 
 ```ts
-import type { AttributifyAttributes } from '@unocss/preset-attributify'
+declare namespace svelteHTML {
+	import type { AttributifyAttributes } from '@unocss/preset-attributify'
```

Comment:
```suggestion
  import type { AttributifyAttributes } from '@unocss/preset-attributify'
```

Reply:
My bad for being lazy and using the GitHub editor instead of just cloning and using VSCode 😅 should be ok now